pkg/ds/farm_test.go

```diff
@@ -506,7 +506,7 @@ func TestFarmSchedule(t *testing.T) {
 	dsID = labeled.Labels.Get(DSIDLabel)
 	Assert(t).AreEqual(anotherDSData.ID.String(), dsID, "Unexpected dsID labeled")
 
-	dsStore.Delete(anotherDSData.ID)
+	err = dsStore.Delete(anotherDSData.ID)
```

Comment:
that was a line I added? 

yeah it was.

I am sorry. The errors are completely bogus because I didn't assign too...